### PR TITLE
Add new permissions

### PR DESCRIPTION
**Pull request by Puncher1**
## Summary

This PR adds following new permissions:
* `manage_guild_expressions` -> `manage_emojis` and `manage_emojis_and_stickers` are now both aliases. This also updates the classmethods `all_channel()` and `elevated()`
* `use_soundboard` -> This also updates the classmethods `all()`, `all_channel()`, `voice()`

Refer to the related PR: https://github.com/discord/discord-api-docs/pull/6017

## Checklist

- [x] If code changes were made then they have been tested.
    - [x] I have updated the documentation to reflect the changes.
- [ ] This PR fixes an issue.
- [x] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [ ] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

File: discord/permissions.py [hunk 1]
```diff
@@ -221,6 +221,9 @@ def all_channel(cls) -> Self:
            Added :attr:`create_public_threads`, :attr:`create_private_threads`, :attr:`manage_threads`,
            :attr:`use_external_stickers`, :attr:`send_messages_in_threads` and
            :attr:`request_to_speak` permissions.
+
+        .. versionchanged:: 2.3
+            Replaced :attr:`manage_emojis` with :attr:`manage_guild_expressions`
```
Comment (Rapptz) [inline discord/permissions.py:226]: No need to mention this since it's a transparent alias.

File: discord/permissions.py [hunk 2]
```diff
@@ -305,11 +308,14 @@ def elevated(cls) -> Self:
...
         - :attr:`manage_webhooks`
-        - :attr:`manage_emojis_and_stickers`
+        - :attr:`manage_guild_expressions`
         - :attr:`manage_threads`
         - :attr:`moderate_members`
 
         .. versionadded:: 2.0
+
+        .. versionchanged:: 2.3
+            Replaced :attr:`manage_emojis_and_stickers` with :attr:`manage_guild_expressions`
```
Comment (Rapptz) [inline discord/permissions.py:318]: No need to mention this since it's a transparent alias.

File: discord/permissions.py [hunk 3]
```diff
@@ -544,13 +550,22 @@ def manage_webhooks(self) -> int:
...
+
+        .. versionadded:: 2.3
+        """
+        return 1 << 30
+
+    @make_permission_alias('manage_guild_expressions')
     def manage_emojis(self) -> int:
-        """:class:`bool`: Returns ``True`` if a user can create, edit, or delete emojis."""
+        """:class:`bool`: An alias for :attr:`manage_guild_expressions`."""
+
```
Comment (Rapptz) [inline discord/permissions.py:563]:
```suggestion
```

Unnecessary whitespace.

**Review by Rapptz — CHANGES_REQUESTED** [review]
This is missing the changes in `all()` and the other classmethods.

In the future there will be `create_guild_expression` at `1 << 43` and `create_events` at `1 << 44` but those should probably not be in this PR since they're unreleased. For that matter these aren't either.

**Comment by Puncher1** [comment]
Yeah I didn't want to add the `create_guild_expression` and `create_events` since they're unreleased. `use_soundboard` is, at least in the beta client. `VIEW_CREATOR_MONETIZATION_ANALYTICS` is also mentioned in the original PR but I can't confirm this.

**Comment by Puncher1** [comment]
> This is missing the changes in `all()` and the other classmethods.
Done, I also added it to `voice()`
